lib/components/InputFieldLabel/index.js

```diff
@@ -0,0 +1,23 @@
+// InputFieldLabel Component
+// Common form field label styles/functionality.
+// Note: This is intended for reuse by other perts-ui components.
+
+import styled from 'styled-components';
+import theme from '../theme';
+
+const InputFieldLabel = styled.label`
+  display: inline-block;
+  width: 100%;
+
+  margin-bottom: ${theme.units.fieldPaddingInternal};
+
+  font-weight: 700;
+  font-size: 0.9rem;
```

Comment:
please-comment: I'm not super well acquainted with the art of css units, so I would find an comment on this choice helpful.

> While em is relative to the font size of its direct or nearest parent, rem is only relative to the html (root) font-size. I like to think of it as a reset. If a style sheet is built in a modular fashion, then rem shouldn’t be needed very often, but it can be handy at times.

https://j.eremy.net/confused-about-rem-and-em/

I could see the choice of `rem` being correct here; you never want the input label to be too small, as might happen if you nested a few `0.9em;`'s.

Reply:
fe2cf8b

Reply:
Added a link to the article. Yes. I was trying to move towards being able to set font-size once, in the html style rule, and scale off that. Won't be 100%, but my hope was that if we wanted to increase or decrease it'd result in less changes later. :shrug: We'll see though.

Reply:
Removed the font-size rule in card since it was using `18px` which is root `font-size`. Leftover copy pasta from triton.

Reply:
lol © 🍝